rulesets/nixpkgs/no-creation.json

```diff
@@ -18,7 +18,8 @@
         "refs/heads/wip-*/**/*",
         "refs/heads/revert-*/**/*",
         "refs/heads/dependabot/**/*",
-        "refs/heads/gh-readonly-queue/**/*"
+        "refs/heads/gh-readonly-queue/**/*",
+        "refs/heads/create-pull-request/**/*"
```

Comment:
We might use this namespace for periodic merges, too, if we move them to PRs. So "PR" in general seems good to me.

Our own names are much shorter in general. (the `gh-readonly-queue` thing is given by GitHub).

I'd prefer something like `pr-` or `pr/**/*`, which has the same meaning.